### PR TITLE
WIP: Keycloak replacement

### PR DESCRIPTION
**Pull request by slawosz**
Before merge
- [x] @nasirkhanbeis to make sure all works
- [x] @ralph-hawkins to make sure all wording is spot on
- [x] @tobyhughes or Bruce to make sure it looks good
- [x] other developer will check for obvious pitfals

### CODE REVIEW

**Comment by nasirkhanbeis** [comment]
Found few issues: 
1.Validation messages on registration form not in order
image

2. Reset pwd email subject - to  be cosmetics specific  
image

3. Remove redundant flash message on reset password confirmation 
image


4. Remove redundant flash message on forgot password email submission
image
5. Create new password page , styling issue. may be we should match like in psd
image

File: cosmetics-web/app/models/organisation.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ class Organisation < ActiveHash::Base
 
   def self.load(force: false)
```
Comment (slawosz) [inline cosmetics-web/app/models/organisation.rb:10]: TODO: delete?

**Review by scruti — APPROVED** [review]
Feature branch ready to be included into master after lots of changes have been merged into this.

There are things that could be improved, some TODOs, some ongoing work based on this... 
But instead of keep feeding the monster, and as the required features work as expected... merging it into master will allow us to keep progressive feature interactions with small PRs.

**Review by scruti — APPROVED** [review]
Feature branch ready to be included into master after lots of changes have been merged into this.

There are things that could be improved, some TODOs, some ongoing work based on this... 
But required feature works as expected and have been approved by QA.
Merging it into master will allow us to keep progressive feature interactions with small PRs.

**Pull request merged**